Makefile

```diff
@@ -8,7 +8,7 @@ VERSION?=$(shell cat VERSION | tr -d " \t\n\r")
 
 FIRST_GOPATH:=$(firstword $(subst :, ,$(shell go env GOPATH)))
 CONTROLLER_GEN_BINARY := $(FIRST_GOPATH)/bin/controller-gen
-CRD_OPTIONS ?= "crd:preserveUnknownFields=false"
+CRD_OPTIONS ?= "crd:crdVersions=v1,preserveUnknownFields=false"
```

Comment:
`preserveUnknownFields=false` is the default in v1, so it should be ok to remove that from the setting if we want.

Reply:
indeed, good catch :+1: